src/components/stage/stage.styles.scss

```
@@ -231,31 +251,41 @@ $animation-easing-title: cubic-bezier(0, 0, 0.5, 1);
+		--stage-logo-base: 38px;
+		--stage-logo-size: var(--stage-logo-base);
+
 		display: flex;
 		flex-wrap: wrap;
 		flex: 1 1 0;
 		align-items: flex-end;
 		align-content: flex-end;
 		justify-content: center;
-		width: 335px;
-		margin: 0 auto $grid-base-five;
+		// Twice the design file's 14/18px: at the file's gap the seven marks
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)